roms_tools/setup/datasets.py

```diff
@@ -128,31 +142,39 @@ def load_data(self) -> xr.Dataset:
             )
 
         # Load the dataset
-        with dask.config.set(**{"array.slicing.split_large_chunks": False}):
-            # Define the chunk sizes
-            chunks = {
-                self.dim_names["latitude"]: -1,
-                self.dim_names["longitude"]: -1,
-            }
-            if "depth" in self.dim_names.keys():
-                chunks[self.dim_names["depth"]] = -1
-            if "time" in self.dim_names.keys():
-                chunks[self.dim_names["time"]] = 1
-
-                ds = xr.open_mfdataset(
-                    self.filename,
-                    combine="nested",
-                    concat_dim=self.dim_names["time"],
-                    coords="minimal",
-                    compat="override",
-                    chunks=chunks,
-                    engine="netcdf4",
-                )
-            else:
-                ds = xr.open_dataset(
-                    self.filename,
-                    chunks=chunks,
-                )
+        if self.use_dask:
+            with dask.config.set(**{"array.slicing.split_large_chunks": False}):
+                # Define the chunk sizes
+                chunks = {
+                    self.dim_names["latitude"]: -1,
+                    self.dim_names["longitude"]: -1,
+                }
+                if "depth" in self.dim_names.keys():
+                    chunks[self.dim_names["depth"]] = -1
+                if "time" in self.dim_names.keys():
+                    chunks[self.dim_names["time"]] = 1
+
+                    ds = xr.open_mfdataset(
+                        self.filename,
+                        combine="nested",
+                        concat_dim=self.dim_names["time"],
+                        coords="minimal",
+                        compat="override",
+                        chunks=chunks,
+                        engine="netcdf4",
```

Comment:
Is there a reason why we pass the `engine` kwarg in one case but not the other?

Reply:
Removed the `engine` kwarg in `1df2b65`.